### PR TITLE
Remove session type

### PR DESCRIPTION
**Pull request by millerts**

### CODE REVIEW

File: client/src/test/java/tds/session/SessionTest.java [hunk 1]
```diff
@@ -85,16 +83,6 @@ public void shouldCrreateAGuestSession() {
     }
 
     @Test
-    public void shouldCreateAGuestSessionCaseInsensitive() {
```
Comment (jtreuting) [inline client/src/test/java/tds/session/SessionTest.java:88]: This doesn't appear to be related to the session type change.  Was this supposed to be removed?

Comment (millerts) [reply client/src/test/java/tds/session/SessionTest.java:88]: Added it back in.
